recipe/build_static.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+_buildd_static=build-static
+_buildd_shared=build-shared
+if [[ ${DEBUG_PY} == yes ]]; then
+  DBG=d
+else
+  DBG=
+fi
+VER=${PKG_VERSION%.*}
+VERABI=${VER}${DBG}
+
+
+cp -pf ${_buildd_static}/libpython${VERABI}.a ${PREFIX}/lib/libpython3.8.a
```

Comment:
The 3.8 here probably shouldn't be explicit

Reply:
Fixed and CI passed successfully.